packages/code-du-travail-frontend/src/common/__tests__/Feedback.test.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { render } from "react-testing-library";
+
+import { Feedback } from "../Feedback";
+
+jest.mock("react-piwik", () => {
```

Comment:
Est-ce qu'on aurait pas intérêt à le mettre dans le setup jest celui là ?

Reply:
je préfére mocker par tests plutôt que globalement car ca permet d'etre explicite sur les dépendance dans un service externe. 
D'ailleurs a terme, je pense qu'il faudrait créer un pub / sub entre l'app et les services externes pour éviter d'avoir un Single Point of failure si un service externe n'est plus joignable

Reply:
En fait je me dis que sur le long terme on va avoir plein de components qui auront piwik en dépendance et que ça va faire beaucoup de dup de code mais bon, pas d'urgence.